requirements.in

```diff
@@ -1,6 +1,6 @@
-click==7.0
+click
 dj-database-url==0.5.0
-django-admin-sortable2
+django-admin-sortable2==1.0.4
 django-autoslug==1.9.4
 django-click==2.1.0
```

Comment:
@amakarudze `django-click==2.2.0` should fix the tests.

Reply:
asjfkhasfkjahfkjfhskjfhjk

Reply:
`django-click==2.2.0` requires `click` to be `>= 7.1` and `< 7.2`. The current `pip-tools` version in `requirements.in` requires `click` to be `>=8`. I'm going to bump `django-click` to `2.3.0` so it also accepts `click>=8`.